src/main/java/Duke.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
```

Comment:
It would be better to state what exactly you are importing instead of .* for example import java.util.Scanner so we know what exactly it is that you imported and will be using